d/dracut/stone.yaml

```diff
@@ -24,6 +24,12 @@ rundeps     :
     - binary(cpio)
     - terminus-font
 setup       : |
+    %patch %(pkgdir)/patches/downstream/0001-Add-module-to-invalidate-depmod-index.patch
+    %patch %(pkgdir)/patches/downstream/0001-Support-stateless-glibc-ldconfig-configuration.patch
+    %patch %(pkgdir)/patches/downstream/0003-Exclude-unwanted-paths-from-initrd-creation.patch
+    %patch %(pkgdir)/patches/downstream/0004-Downstream-De-compress-kbd-files.patch
+    %patch %(pkgdir)/patches/downstream/0001-Allow-for-skipping-firmware-installation.patch
```

Comment:
Also seems Solus specific, would be nice to get an upstream bug in dracut-ng to track progress

Reply:
Not really a point tbh I'm not sure of any distro that is taking this approach with the initrd.